docs/toml.md

```diff
@@ -328,6 +326,65 @@ To write JSON format logs, specify `json` as the format:
 # interval = "30s"
 ```
 
+## Responding timeouts
+```
+# Timeout settings for the http servers Traefik starts
```

Comment:
I improved the sentence. Please check again whether it is more clear now for you.